app/models/wiki_content/revision.rb

```diff
@@ -66,4 +67,11 @@ def plagiarism_report_link
     return unless ithenticate_id
     "/recent-activity/plagiarism/report?ithenticate_id=#{ithenticate_id}"
   end
+
+  def references_added
+    current_refs = features['feature.wikitext.revision.ref_tags']
+    prev_refs = features_previous['feature.wikitext.revision.ref_tags'] || 0
+    return 0 if current_refs.nil? || current_refs < prev_refs
```

Comment:
Negative value for references added is okay; it should return the negative number instead of zero in that case.